data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+	invisible
+	landing
+	name "Wanderers: Professor Andelena"
+	source Allhome
+	to offer
+		has "Wanderers: Professor Available: done"
+		not "Wanderers: Sayari Available After Hai Reveal: offered"
+	to complete
+		has "Wanderers: Diplomacy (Before Hai Reveal): offered"
+	on offer
+		log "Minor People" "Professor Andelena" `Professor Maindell Stormchild Andelena, IV speaks with Shroud cloud city accent and is a daughter of the rich Andelena family. Her wealthy meanderings have led her to a university on Allhome where she studies Skiorology, the Hai-studying equivalent of Anthropology.`
+		log "Minor People" "Eenap" `Professor Andelena's small Hai grad student, who is interested in studying the aliens to the north as her thesis research subject.`
+		conversation
+			`It turns out Mongongo University is the size of a large town, with its own spaceport. The Department of Skiourology has five entire buildings. You enter the largest and inquire about Professor Andalena, whose office is through a labyrinth of corridors. After a few minutes of navigation, you begin to conclude you've been trapped in a Mobius strip.`
+			`	"Captain <last>," squeaks a Hai voice near you. "I'm Eenap, Professor Andelena's grad student. Please follow me to her office so we may discuss an expedition." Soon, you arrive at Professor Andelena's office, which looks like a small museum exhibit, filled with taxidermy statues of evolutionary ancestors of the Hai, and diagrams of development of Hai society.`
+			`	"Welcome, Captain <last>! I am Professor Maindell Stormchild Andelena, IV." This human woman speaks with the famously-dignified Shroud cloud city accent. "We were planning an expedition to study planets suspected of being part of a past Hai empire, but your discovery has piqued our interest."`
 			choice
-				`	"Indeed, I'm hoping you have someone in mind."`
-				`	"It would be great to work with you again if you're willing."`
-			`	You open your mouth to respond but the effort is pointless.`
-			`	"I accept!" she cries, leaning forward on the desk with a gleeful look in her eyes.`
-			`	"A first contact is a dream opportunity, and there are so very few of them. So of course I will come with you."`
+				`	"You said you were... Skiorologists?"`
+				`	"My only interest is translation. Will you do that for me?"`
+					goto "get to the point"
+			`	"As Skiorologists, we study the development of the Hai, in the same way that Anthropologists study humans. How the Hai interact with other species is a major part of what makes the Hai... Hai. We have researched past interactions, but analyzing an ongoing interaction is a dream come true.`
+			label "get to the point"
+			`	"In return for introducing us to this new species, we'll be happy to translate for you. As we're already prepared to leave for an expedition, we can leave as soon as you meet us in the spaceport."`
 			choice
-				`	"Let's go visit these aliens immediately!"`
+				`	"Sure! I'll meet you at the spaceport."`
```

Comment:
```suggestion
				`	"Sure! I'll meet you at my ship."`
```

Reply:
This is because of the `accept`. If you actually do meet her in the spaceport bar or something like that, you can ignore this.

Reply:
It isn't her ship. It's your ship. The standard in Endless Sky is to meet "at the spaceport." It's what the player is trained to expect for spaceport missions.

Reply:
If you like. This can be resolved.